static/app/components/feedback/list/feedbackListWrapper.tsx

```diff
@@ -0,0 +1,27 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import PanelItem from 'sentry/components/panels/panelItem';
+import {space} from 'sentry/styles/space';
+
+export default function FeedbackListWrapper({children}) {
+  return (
+    <Fragment>
+      <HeaderPanelItem>fixed header</HeaderPanelItem>
+      <OverflowPanelItem style={{padding: space(0.5)}}>{children}</OverflowPanelItem>
```

Comment:
Why is this inlined?

Reply:
That is me being lazy. I'll get rid of it. 

Also, there are two things called `OverflowPanelItem`, and this is the only difference. Maybe i can shove this padding on another component instead.